### DISCUSSION TITLE
Add a slashcommand flag for 'view' which allows opening a record or list in the specified view name

### DISCUSSION THREAD

**Discussion by VanWeapon** [arnoudkooi/SN-Utils#303] [Ideas]
Examples
/incident -v mim 
Should open the incident list in the "mim" view

/sys_dictionary -n -v advanced 
should open a new dictionary record in the advanced view

I realised while typing this that none of the existing flags allow you to provide parameters as input to the flag, so maybe this isn't the ideal way to implement it.

**Comment by VanWeapon** [comment]
Okay i've accomplished the new record version with a slashcommand:
![image](https://user-images.githubusercontent.com/30616047/190943983-22b4f78d-d663-4e9c-b732-bc18e5f0a440.png)

**Reply by arnoudkooi** [reply]
Nice!
